management/src/main/java/org/kaazing/gateway/management/config/ServiceDefaultsConfigurationBeanImpl.java

```diff
@@ -119,6 +119,16 @@ public String getAcceptOptions() {
                     jsonOptions.put("http.transport", httpTransport);
                 }
 
+                Long tcpHandshakeTimeout = (Long) acceptOptions.remove("tcp.handshake.timeout");
+                if (tcpHandshakeTimeout != null) {
+                    jsonOptions.put("tcp.handshake.timeout", tcpHandshakeTimeout);
+                }
+
+                Long httpHandshakeTimeout = (Long) acceptOptions.remove("http.handshake.timeout");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "http.handshake.timeout" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AS1192)